src/ExtUvLoop.php

```diff
@@ -0,0 +1,320 @@
+ *
+ * This loop is known to work with PHP 7+.
+ *
+ * @see https://github.com/bwoebi/php-uv
+ */
+final class ExtUvLoop implements LoopInterface
+{
+    private $uv;
+    private $futureTickQueue;
+    private $timers;
+    private $streamEvents = array();
+    private $readStreams = array();
+    private $writeStreams = array();
+    private $running;
+    private $signals;
+    private $signalEvents = array();
+    private $streamListener;
+
+    public function __construct()
+    {
+        if (!\function_exists('uv_loop_new')) {
+            throw new \BadMethodCallException('Cannot create LibUvLoop, ext-uv extension missing');
+        }
+
+        $this->uv = \uv_loop_new();
+        $this->futureTickQueue = new FutureTickQueue();
+        $this->timers = new SplObjectStorage();
+        $this->streamListener = $this->createStreamListener();
+        $this->signals = new SignalsHandler();
+    }
+
+    /**
+     * Returns the underlying ext-uv event loop. (Internal ReactPHP use only.)
+     *
+     * @internal
+     *
+     * @return resource
+     */
+    public function getUvLoop()
+    {
+        return $this->uv;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function addReadStream($stream, $listener)
+    {
+        if (isset($this->readStreams[(int) $stream])) {
+            return;
+        }
+
+        $this->readStreams[(int) $stream] = $listener;
+        $this->addStream($stream);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function addWriteStream($stream, $listener)
+    {
+        if (isset($this->writeStreams[(int) $stream])) {
+            return;
+        }
+
+        $this->writeStreams[(int) $stream] = $listener;
+        $this->addStream($stream);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function removeReadStream($stream)
+    {
+        if (!isset($this->streamEvents[(int) $stream])) {
+            return;
+        }
+
+        unset($this->readStreams[(int) $stream]);
+        $this->removeStream($stream);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function removeWriteStream($stream)
+    {
+        if (!isset($this->streamEvents[(int) $stream])) {
+            return;
+        }
+
+        unset($this->writeStreams[(int) $stream]);
+        $this->removeStream($stream);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function addTimer($interval, $callback)
+    {
+        $timer = new Timer($interval, $callback, false);
+
+        $that = $this;
+        $timers = $this->timers;
+        $callback = function () use ($timer, $timers, $that) {
+            \call_user_func($timer->getCallback(), $timer);
+
+            if ($timers->contains($timer)) {
+                $that->cancelTimer($timer);
+            }
+        };
+
+        $event = \uv_timer_init($this->uv);
+        $this->timers->attach($timer, $event);
+        \uv_timer_start(
+            $event,
+            (int)($interval * 1000) + 1,
```

Comment:
Shouldn't these two lines have a space between the type cast and the operation, just like all the other type casts here?

Reply:
Yup, updated it